django-stubs/contrib/gis/db/backends/mysql/features.pyi

```diff
@@ -14,7 +15,7 @@ class DatabaseFeatures(BaseSpatialFeatures, MySQLDatabaseFeatures):
     supports_num_points_poly: bool
     @property
     def empty_intersection_returns_none(self) -> bool: ...
-    @property
+    @cached_property
     def supports_geometry_field_unique_index(self) -> bool: ...  # type: ignore[override]
-    @property
+    @cached_property
     def django_test_skips(self) -> dict[str, Any]: ...  # type: ignore[override]
```

Comment:
Doesn't seem to exist in the GIS DatabaseFeatures class, it's in the ordinary MySQL DatabaseFeatures class.

Looks like something that can be moved in a separate PR if you're up for it.

Reply:
They existed before 4.2 (See [4.1 for ex](https://github.com/django/django/blob/stable/4.1.x/django/contrib/gis/db/backends/mysql/features.py) ).

I think it fine to remove them because they are `Classvar` in base classes:
- `django_test_skips` -> https://github.com/django/django/blob/stable/4.2.x/django/db/backends/base/features.py#L366C20-L366C20
- `empty_intersection_returns_none` -> https://github.com/django/django/blob/stable/4.2.x/django/contrib/gis/db/backends/base/features.py#L59

I'll update base types in a separate PR